tests/modules/nf-scil/registration/tractogram/main.nf

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { LOAD_TEST_DATA } from '../../../../../subworkflows/nf-scil/load_test_data/main.nf'
+include {
+    REGISTRATION_TRACTOGRAM as REGISTRATION_TRACTOGRAM_1; } from '../../../../../modules/nf-scil/registration/tractogram/main.nf'
```

Comment:
Why the 1 at the end of your module's name ? You don't need to rename the module here, just import it.